iosHyperskillApp/iosHyperskillApp/Sources/Modules/StudyPlan/Views/StudyPlanView.swift

```diff
@@ -78,8 +78,12 @@ struct StudyPlanView: View {
                             .foregroundColor(.secondaryText)
                             .padding(.bottom, appearance.trackTitleBottomPadding)
                     }
-                    // TODO: ALTAPPS-701 change problems limit embedding
-//                    ProblemsLimitAssembly().makeModule()
+
+                    ProblemsLimitView(
+                        stateKs: viewModel.problemsLimitViewStateKs,
+                        onReloadButtonTap: viewModel.doReloadProblemsLimit
+                    )
+                    .padding(.top, LayoutInsets.smallInset)
```

Comment:
The track title already has a bottom padding `.padding(.bottom, appearance.trackTitleBottomPadding)`. Could you please attach here a screenshot of how the study plan looks like when the problems limit view is visible with top padding?

Reply:
Yep, you're right, it was too much. Now it seems ok:
![Simulator Screen Shot - iPhone 14 Pro - 2023-05-12 at 17 26 25](https://github.com/hyperskill/mobile-app/assets/30647936/122f5e2f-66da-46a4-863c-4c4335c5b062)
![Screenshot_20230512_172302](https://github.com/hyperskill/mobile-app/assets/30647936/7af407fd-9ac7-46e6-bb0c-1e91f507e5d5)

Reply:
On Android looks nicer